docs/flow/reference/components/prometheus.integration.node_exporter.md

```diff
@@ -0,0 +1,315 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/prometheus.integration.node_exporter
+title: prometheus.integration.node_exporter
+---
+
+# prometheus.integrations.node_exporter
+The `prometheus.integrations.node_exporter` embeds 
+[node_exporter](https://github.com/prometheus/node_exporter) which exposes a 
+wide variety of hardware and OS metrics for \*nix-based systems.
+
+The `node_exporter` itself is comprised of various _collectors_, which can be
+enabled and disabled at will. For more information on collectors, refer to the
+[`collectors-list`](#collectors-list) section.
+
+The `prometheus.integrations.node_exporter` component is a _singleton_
+component. That means that it can only appear once per configuration file, and
+a block label must not be passed to it.
+
+## Example
+```river
+prometheus.integrations.node_exporter {
+}
+
+// Configure a prometheus.scrape component to collect node_exporter metrics.
+prometheus.scrape "demo" {
+  targets    = prometheus.integrations.node_exporter.targets
+  forward_to = [ /* ... */ ]
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`set_collectors`           | `list(string)` | Overrides the default set of enabled collectors with the collectors listed. | | no
+`enable_collectors`        | `list(string)` | Collectors to mark as enabled.  | | no
+`disable_collectors`       | `list(string)` | Collectors to mark as disabled. | | no
+`include_exporter_metrics` | `boolean`      | Whether metrics about the exporter itself should be reported. | false | no 
+`procfs_path`              | `string`       | The procfs mountpoint. | `/proc` | no
+`sysfs_path`               | `string`       | The sysfs mountpoint.  | `/sys`   | no
+`rootfs_path`              | `string`       | Specify a prefix for accessing the host filesystem. | `/` | no
+
+`set_collectors` defines a hand-picked list of enabled-by-default
+collectors. If set, anything not provided in that list is disabled by
+default.
```

Comment:
Can we link to the bottom of the page so people know which collectors are enabled by default for their system?

```markdown
See [Collectors list](#collectors-list) for the default set of enabled collectors for each supported system. 
```